c7n/resources/ses.py

```diff
@@ -0,0 +1,32 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+from c7n.manager import resources
+from c7n.query import DescribeSource, QueryResourceManager, TypeInfo
+from c7n.utils import local_session
+from c7n.tags import universal_augment
+
+class DescribeConfigurationSet(DescribeSource):
+
+    def augment(self, resources):
+        client = local_session(self.manager.session_factory).client('ses')
+        for r in resources:
+            details = client.describe_configuration_set(ConfigurationSetName=r['Name'], 
+                ConfigurationSetAttributeNames=['eventDestinations','trackingOptions','deliveryOptions','reputationOptions'])
+            r.update(details)
```

Comment:
I think we can ditch a couple keys here. We already have the configuration set name, and we generally don't want to record the response metadata.

```suggestion
            r.update({
                k: details[k]
                for k in details
                if k not in {'ConfigurationSet', 'ResponseMetadata'}
            })
```

Reply:
oh good point, yup completely missed that.